library-core/src/main/java/com/mikepenz/fastadapter/adapters/ItemFilter.kt

```diff
@@ -55,15 +46,7 @@ open class ItemFilter<Model, Item : GenericItem>(private val mItemAdapter: Model
      */
     open val selectedItems: Set<Item>
         get() {
-            return originalItems?.let { originalItems ->
```

Comment:
I did this implementation on purpose, because filter.toSet does two allocations while doing it manually only allocates a hashset instead of a list and a hashset.